server/src/main/java/com/cloud/api/ApiResponseHelper.java

```diff
@@ -3991,7 +3991,7 @@ public UsageRecordResponse createUsageResponse(Usage usageRecord, Map<String, Se
                 final StringBuilder builder = new StringBuilder();
                 builder.append("VMSnapshot on primary storage usage");
                 if (vmSnapshotVO != null) {
-                    builder.append(" Id: ").append(vmSnapshotVO.getId()).append(" (").append(vmSnapshotVO.getUuid()).append(") ");
+                    builder.append(" Id: ").append(vmSnapshotVO.getUuid()).append(" (").append(vmSnapshotVO.getUuid()).append(") ");
```

Comment:
`Id: <uuid>` should be ok. no need to keep uuid twice `Id: <uuid> (<uuid>)`

Reply:
we may need the same change at line:3951 https://github.com/apache/cloudstack/blob/56ef349b0e49d00d21c63d2c2ef7bee6669e82a2/server/src/main/java/com/cloud/api/ApiResponseHelper.java#L3951